docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -185,61 +181,87 @@ repo_gpgcheck=1
 </TabItem>
 </Tabs>
 
-2. Run `sudo yum update` to refresh your repo data cache. Optionally, you may wish to also install all available updates.
+2. Run `sudo yum update` to refresh the repo data cache.
 2. Run `sudo yum install ziti-edge-tunnel` to install the RPM.
-2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
-2. Set the filemode and owner so that members of group `ziti` will be able to read and delete the token file.
-
-  ```text
-  sudo chown -cR :ziti        /opt/openziti/etc/identities
-  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
-  ```
-
 2. Enable and start the service
 
     ```text
     sudo systemctl enable --now ziti-edge-tunnel.service
     ```
 
-2. The process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+2. Add an Identity
 
-    ```text
-    sudo systemctl restart ziti-edge-tunnel.service
-    ```
+## Adding an Identity
+
+### Restart to Load All in Identity Directory
+
+The tunneler will load all identities in the `--identity-dir` directory. The default location for identities is is
+`/opt/openziti/etc/identities`. Add identity files to this directory by copying the identity file into the directory and
+setting permissions for group `ziti` so the tunneler service may delete JWT enrollment tokens and read and write the
+JSON identity configuration files.
+
+```text
+sudo chown -cR :ziti        /opt/openziti/etc/identities
+sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
+```
+
+The tunneler process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+
+```text
+sudo systemctl restart ziti-edge-tunnel.service
```

Comment:
Since this page has both manual and package based installation, this section should have a caveat on it depending on how the user started the tunneler. soemthing like:

"If the tunneler was installed using systemd, restart the process with: systemctl restart ziti-edge-tunnel. "